models/iaf_psc_alpha.h

```diff
@@ -96,10 +96,11 @@ Name: iaf_psc_alpha - Leaky integrate-and-fire neuron model.
   I_e        double - Constant external input current in pA.
   V_min      double - Absolute lower value for the membrane potential.
 
-Remarks:
-  tau_m != tau_syn_{ex,in} is required by the current implementation to avoid a
-  degenerate case of the ODE describing the model [1]. For very similar values,
-  numerics will be unstable.
+Note:
```

Comment:
This should be `Remarks:` not `Note:`.  Only `Remarks` is a keyword in the NEST documentation system.